### ISSUE TITLE
Composite resource single file metadata files appear when an unrelated file is unzipped

### ISSUE THREAD

**Issue by dtarb** [hydroshare/hydroshare#2833]
In the 1.16 release candidate deployed on https://dev-hs-6.cuahsi.org I did the following
1. Added file (Readme.txt)
2. Switched to edit mode to set metadata for that file.  This converted the file to a Single file aggregation
3. Added an unrelated zip file (logan_vrt_small.zip)
4. Unzipped the zip file.
5. The content display now shows in addition to the folder from the zip file, the two metadata files from the single file content type aggregation readme.txt.  This should not be the case.  *meta.xml and *resmap.xml should be hidden, and they were until the unrelated zip file is unzipped.
![image](https://user-images.githubusercontent.com/3456833/42717555-164cf6e4-86bf-11e8-96fa-76340a325270.png)

The resource where I showed this is https://dev-hs-6.cuahsi.org/resource/4245b73c5cb54111b276dea5239f755b/ (but this will disappear next time hs-6 is rebuilt.

An additional aspect of this problem is that if after step 2, I put /debug after the resource URL I get the following errors.
![image](https://user-images.githubusercontent.com/3456833/42717738-5b733b9c-86c0-11e8-9643-51f51ed9a968.png)

This leads me to interpret that the display or hiding of metadata files is being managed by whether they are added to Django or not.  This does not seem to be a reliable approach.

**Comment by dtarb** [comment]
@alvacouch perhaps you could just point to the management command functions that need updating and @aphelionz @phuongdm or @sblack-usu could make the changes.

**Comment by pkdash** [comment]
I have made a fix to the unzip function so that it does not add the aggregation xml files to Django. I can create a PR tonight. That may help in modifying the management commands.

**Comment by pkdash** [comment]
@alvacouch Please check my last commit (see above). I am assuming that should fix the debug_resource.

**Comment by sblack-usu** [comment]
Could I get some others to weigh in on the implementation here?  While making the changes to unzip files to a new folder, I also made it so the folder index would increment to prevent overwriting existing files.  (https://github.com/hydroshare/django_irods/pull/44#discussion_r203739444)

Turns out there are a couple of tests out there that I didn't see which tests for the action to be blocked if files are overwritten.  (http://ci.hydroshare.org:8080/job/hydroshare-pull-requests/1741/testReport/hs_core.tests/TestResourceFileFolderOprsAPI/test_resource_file_folder_oprs/)

I can update this PR to be compliant with these tests, but I thought I'd get some input before updating the PR.

**Comment by aphelionz** [comment]
@sblack-usu What's the status of this?

**Comment by sblack-usu** [comment]
it's been merged into the main composite resource branch

**Comment by dtarb** [comment]
Fixed with 1.16.
